pkgs/top-level/stage.nix

```diff
@@ -126,7 +127,8 @@ let
     trivialBuilders
     splice
     allPackages
-    aliases
+    (if (config.quickEval or false) then (self: super: {}) else defaultOverrides)
```

Comment:
@nbp Does this flag make sense to you? My main goal is to get rid of overrides and aliases in Nixpkgs. Making their overlays optional seems like a straightforward way to do that (of course it will break eval currently, but give us a way to find where their used).

Reply:
You are interested in drawing the graph of dependencies between various attributes, in order to remove the unused one?

I think this might make sense for the "soon to be removed", in which case this could give a grace period for people to move these to their own overlays.  Maybe `config.enableObsoleteOverrides` would be more explicit.

Otherwise to detect which one can be moved to this list, I suggest to make an overlay which does something like:

```nix
self: super:

super.lib.mapAttrs (n: v: __trace n) super
```

Then evaluate the whole list of packages you are interested in, and pipe the error output into `sort -Ru` which should give you the list of top-level packages / overrrides which are used directly or indirectly.